episodes/introduction.md

```diff
@@ -1,28 +1,213 @@
+If your project requires large quantities of data, sufficiently large that your computer cannot load it all into memory at once, you will need to use techniques and tools that are built to handle data at scale.
+
+<!-- Throughout the process, you may wish to get feedback from your peers and mentors about your progress and approach. -->
+<!-- There are many steps you can take to make collaboration easier technologically as well as ensuring your work is legible to others. -->
+
+The parts of the research project life cycle that typically receive explicit attention in coursework include selecting a research question, analysis techniques (e.g., selecting the appropriate statistical method), and publication.
+Training in the remaining portions is often assumed to happen naturally through research experience, but this is not always effective in practice.
+Gaps in these areas can create roadblocks to conducting effective, reproducible and open research, even for experienced researchers.
+This course aims to address some of those roadblocks.
+
+:::::::: challenge
+
+#### Introspection: untaught skills
+
+Think of a skill you've found useful in your research that wasn't taught in one of your classes. How did you learn it? Have you taught it to anyone else? Why or why not?
+
+::::::::
+
+#### What we will cover
+
+This course is focused on practical solutions to roadblocks you may encounter in dealing with data, code, and collaboration. We will be following the arc of an open data analysis project in Python, structuring the course into three sections:
+
+Roadblocks to data acquisition:
+- My data is in a format I've never worked before
+- The data I want to work with is published through an Application Programming Interface
+(API), and I don't know how to download it
+
+Roadblocks to data cleaning & processing:
+- There's something unexpected about my input data, but I'm not sure what
+- The code runs on some of my data, but errors on other input data
+- When I re-run my code I get different results, and I'm not sure why
+- I have no idea which part of my code is causing a particular problem
+- My data changes format or content over time
+- My data is too big to work with on a desktop computer
+
+Roadblocks to collaboration:
+- I'm not sure how to make it simple for collaborators to run and contribute to my code
+- I need to publish my code and/or data for a paper I'm submitting to, but I'm not sure
+how best to do so
+- A colleague wants to build on my existing code, but I'm not sure how to clearly document
+what I've done to make it possible for them to adapt it
+- I wrote this code myself six months ago, and I do not recognize it, nor can I remember what I was trying to do
+
+The next episode will discuss reading data in unexpected file formats. But first, we have to locate some data that will help with our research goals.
```

Comment:
Transition... whiplash or judo? not sure

Reply:
Maybe there's something here about "finding data that looks suspiciously like it will help with our research goals?"

No matter how good you are at skimming the available metadata/context you will still have to look at the GD data before you can assess whether it's useful. And "looking at the GD data" is a big part of the rest of the course, so this section of the intro is really about "is this even worth looking at?"